apps/nuxt3-ssr/components/SearchResultsCount.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="mt-1 mb-0 lg:mb-3 text-body-lg flex flex-col text-title">
+    <p>{{ value }} {{ pluralizedLabel }}</p>
+  </div>
+</template>
+
+<script setup lang="ts">
+interface Props {
+  value: Number;
+  label?: String;
+  suffix?: String;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  value: 0,
+  label: "result",
+  suffix: "s",
+});
+
+let pluralizedLabel = computed(() => {
```

Comment:
can be a const , the value is the reactive part , the pointer is not being reassigned